README.md

```diff
@@ -28,6 +30,39 @@ Installation
 composer require biig/domain
 ```
 
+Basic usage
+-----------
+
+```php
+class YourModel extends DomainModel
+{
+    public const CREATION = 'creation';
+    public function __construct()
+    {
+        $this->dispatch(self::CREATION, new DomainEvent($this);
+    }
+}
+```
+
+```php
+class DomainRule extends DomainRuleInterface
```

Comment:
implements

Reply:
You're absolutely right but this will be fixed in another PR/commit because this PR is a backport for another project (and commits will remain the same)